### DISCUSSION TITLE
[Mobile] Is there any method to scroll to some word/offset ?

### DISCUSSION THREAD

**Discussion by sagarkardani** [singerdmx/flutter-quill#346] [Mobile]
I want to find and highlight some words. Probably I would be able to highlight words somehow with controller or via some other methods.

Another thing I want is to jump to/scroll to previous and next instance of found words. I am unable to find any methods like `_controller.scrollToPosition()` or `_controller.scrollToOffset()`. Is there any such methods available in controller ?

Below is the screenrecording of expected behaviour.

https://user-images.githubusercontent.com/60609139/129385745-6beab317-0be9-44ca-94b7-c0b94cca4a44.mp4

Regards,
Sagar

**Comment by singerdmx** [comment]
You are welcome to submit a PR

**Comment by sagarkardani** [comment]
@singerdmx

Thanks for quick reply.

At present I am just cheking that my other requirements are available in flutter-quill or not. What I have shown in video is achieved via Codemirror in Webview. Ofcourse Codemirror is very powerful, but I want to switch over to flutter-quill to avoid loading Webview and to manage this thing directly via flutter-ui. Whole app is very big, so managing this part directly in flutter will considerably decrease RAM consumption.

If I am able to switch over this thing to flutter-quill, I will surely submit a PR.

Regards,
Sagar.

**Reply by singerdmx** [reply]
Sounds good.
If you need help making the change, just let me know
